custom/icds_reports/management/commands/queue_person_cases.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.models import AsyncIndicator
+from django.db import connections
+
+from dimagi.utils.chunked import chunked
+
+
+class Command(BaseCommand):
+    help = "Rebuild Bihar person cases (Child and AG Cases)"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'type_of_cases',
+            type=str
+        )
+
+    def handle(self, *args, **kwargs):
+        # by default execute child cases
+        type_of_cases = kwargs['type_of_cases'] if kwargs['type_of_cases'] else "child_cases"
+        person_config = _get_config_by_id('static-icds-cas-static-person_cases_v3')
+        # sort by supervisor_id and doc_id to improve the performance, sorting is needed to resume the queueing
+        # if it fails in between.
+        # AG Cases
+        query = ""
+        if type_of_cases == 'ag_cases':
+            # AG CASES
+            table_name = get_table_name('icds-cas', 'static-person_cases_v3')
+            # includes all the valid cases for the march april and may 2020
+            ag_end_range = '01-06-2009'
+            ag_start_range = '01-03-2006'
+            query = f"""
+                select supervisor_id, doc_id from "{table_name}"
+                where state_id='f9b47ea2ee2d8a02acddeeb491d3e175' AND sex='F'
+                AND dob::DATE>'{ag_start_range}' AND dob::DATE<={ag_end_range}
```

Comment:
these comparisons signs do not what is there in agg_Awc. it would be  like
`ag_start_range ≤dob< ag_start_range` no?

Reply:
updated!